### DISCUSSION TITLE
[Script request] FreshRSS

### DISCUSSION THREAD

**Discussion by CrazyWolf13** [community-scripts/ProxmoxVE#31] [Request script] [answered]
### Application Name

FreshRSS

### Website

https://www.freshrss.org/

### Description

FreshRSS is a self-hosted RSS and Atom feed aggregator.
It is lightweight, easy to work with, powerful, and customizable. 

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.

**Comment by MickLesk** [comment, marked as answer]
Done & merged by @bvdberg01
